NOTICE.md

```diff
@@ -1,12 +1,12 @@
 # Bento NOTICE
 
 Chef Bento
-Copyright 2012-2019, Chef Software, Inc.
+Copyright 2012-2022, Progress Software Corporation
```

Comment:
pretty sure you need both copyrights on this

Reply:
thanks.